code/game/atoms.dm

```diff
@@ -1435,8 +1435,13 @@
 		var/list/atom/created_atoms = list()
 		for(var/i = 1 to chosen_option[TOOL_PROCESSING_AMOUNT])
 			var/atom/created_atom = new atom_to_create(drop_location())
-			created_atom.pixel_x = rand(-8, 8)
-			created_atom.pixel_y = rand(-8, 8)
+			if(custom_materials)
+				created_atom.set_custom_materials(custom_materials, 1 / chosen_option[TOOL_PROCESSING_AMOUNT])
```

Comment:
this'll have a bunch of unrelated changes, why do you need this

Reply:
It seems like a less snowflakey solution than adding ownership vars to all the products between meat and burger and then adding special code in situations like this to pass them on. 

Adding vars storing the source's name and job to the mob_meat material seemed like the exact sort of thing you intended the material system to be used for. I was originally going to have everything made of mob_meat name itself after the owner (e.g. "Leroy Coldsmith meat toilet") but there were issues with some of the names.

It seems like an oversight (or unfinished) that materials are passed through the food processor but not through tool processing or grilling. At the moment custom materials from raw meat passes into raw meatballs from the food processor but disappears after the meatball is flattened into a patty or grilled. 

What consequences do you think this will have, anyway? Unless you have something specific in mind I would just wait and see what happens and fix what arises, if anything.